File: Competitive Coding/C/Keshav Jain/Selection Sort.c
#include<stdio.h>
void sel_sort(int a[], int n)
{
    int i, j, min, temp;
    for(i=0 ; i<n-1 ; i++)
    {
        min = i;
        for(j=i+1 ; j<n ; j++)
        {
            if(a[j] < a[min])
                min = j;
        }
        temp = a[min];
        a[min] = a[i];
        a[i] = temp;
    }
}
int main()
{
    int a[50], b[50], n;
    printf("Enter no. of elements: ");
    scanf("%d",&n);
    printf("Enter Array:\n");
    for(int i=0 ; i<n ; i++)
        scanf("%d",&a[i]);
    sel_sort(a, n);
    for(int i=0 ; i<n ; i++)
        printf("%d\n",a[i]);
}

File: Competitive Coding/Java/Anchal Goyal/selectionSort.java
import java.util.Scanner;
// Other imports go here
// Do NOT change the class name
class selectionSort
{
    static void selectionSort(int arr[],int n)
    {
        int min=arr[0];
        int temp=0;
        for(int i=0;i<n;i++)
        {
            min=i;
            for(int j=i+1;j<n;j++)
            {
                if(arr[min]>arr[j])
                {
                    min=j;
                }

            }
            temp=arr[min];
            arr[min]=arr[i];
            arr[i]=temp;
        }
    }
   
    public static void main(String[] args)
    {
        // Write your code here
        Scanner s=new Scanner(System.in);
        System.out.print("enter no of elements");
        int n=s.nextInt();
        int arr[]=new int[n];
        for(int i=0;i<n;i++)
        {
            arr[i]=s.nextInt();
        }
        selectionSort(arr,n);
        for(int i=0;i<n;i++)
        {
            System.out.print(arr[i]+" ");
        }
    }
    }